packages/flutter/test/material/data_table_test.dart

```diff
@@ -94,9 +106,40 @@ void main() {
     ));
     await tester.pumpAndSettle(const Duration(milliseconds: 200));
 
-    await tester.tap(find.text('375'));
+    await tester.tap(find.text('375'),pointer: 2);
```

Comment:
Is there a need for specifying the pointers (2, 3, 4, etc) for this test? I don't think I've ever used them so I'm not familiar with its significance.